docs/installing-solidity.rst

```diff
@@ -262,29 +262,29 @@ You need to install the following dependencies for Windows builds of Solidity:
 +-----------------------------------+-------------------------------------------------------+
 | Software                          | Notes                                                 |
 +===================================+=======================================================+
-| `Visual Studio 2017 Build Tools`_ | C++ compiler                                          |
+| `Visual Studio 2019 Build Tools`_ | C++ compiler                                          |
 +-----------------------------------+-------------------------------------------------------+
-| `Visual Studio 2017`_  (Optional) | C++ compiler and dev environment.                     |
+| `Visual Studio 2019`_  (Optional) | C++ compiler and dev environment.                     |
 +-----------------------------------+-------------------------------------------------------+
 
 If you already have one IDE and only need the compiler and libraries,
-you could install Visual Studio 2017 Build Tools.
+you could install Visual Studio 2019 Build Tools.
 
-Visual Studio 2017 provides both IDE and necessary compiler and libraries.
-So if you have not got an IDE and prefer to develop solidity, Visual Studio 2017
+Visual Studio 2019 provides both IDE and necessary compiler and libraries.
+So if you have not got an IDE and prefer to develop solidity, Visual Studio 2019
 may be a choice for you to get everything setup easily.
 
 Here is the list of components that should be installed
-in Visual Studio 2017 Build Tools or Visual Studio 2017:
+in Visual Studio 2019 Build Tools or Visual Studio 2019:
 
 * Visual Studio C++ core features
-* VC++ 2017 v141 toolset (x86,x64)
+* VC++ 2019 v141 toolset (x86,x64)
 * Windows Universal CRT SDK
 * Windows 8.1 SDK
 * C++/CLI support
 
-.. _Visual Studio 2017: https://www.visualstudio.com/vs/
-.. _Visual Studio 2017 Build Tools: https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2017
+.. _Visual Studio 2019: https://www.visualstudio.com/vs/
+.. _Visual Studio 2019 Build Tools: https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2019
 
 Dependencies Helper Script
```

Comment:
Maybe we should switch the ``Or, on windows:`` part of this to ``scripts/install_deps.ps1`` now? And maybe mention that that doesn't attempt to install dependencies globally, but just puts them in ``deps/``... and that it will not only build boost, but also fetch a cmake version that can be used to ``deps/cmake``...

Reply:
If so we can remove ``scripts/install_deps.bat``...

Reply:
Can you do that, please? :)

Reply:
I just quickly wrote up something at both places, but in a hurry, so better to have another look later I guess.